src/main/java/pl/edu/mimuw/chess/White.java

```diff
@@ -0,0 +1,9 @@
+package pl.edu.mimuw.chess;
+
+public class White extends Player {
```

Comment:
te klasy `White` i `Black` nic nie wnoszą, w sensie zdefiniowania ich (nigdzie nie robisz `getClass()` żeby skorzystać z tego, że to co innego, tylko trzymasz na innycy polach, do tego lepiej dać własność w `Player`) 😞